cosmoz-data-nav.js

```diff
@@ -973,16 +973,21 @@ class CosmozDataNav extends hauntedPolymer('haunted', useDataNav)(PolymerElement
 			return;
 		}
 
+		if (this._readFromHash && !this.maintainSelection) {
+			return;
+		}
+
 		const path = ['_routeHashParams', hashParam],
 			hashValue = this.get(path),
 			selection = this.items.findIndex(i => this._getItemId(i) === hashValue);
 
 		if (selection < 0 || selection === this.selected) {
 			return;
 		}
-
+		this._readFromHash = true;
```

Comment:
Maybe a better name?

```suggestion
		this._readFromHashOnce = true;
```

Reply:
Done!